components/function-controller/internal/controllers/function_test.go.todo

```diff
@@ -0,0 +1,150 @@
+package controllers_test
```

Comment:
remember to delete/rename this file before merge

Reply:
this will be removed once we migrate to ginkgo, thx